docs/fwupd.conf.md

```diff
@@ -61,6 +61,11 @@ The `[fwupd]` section can contain the following parameters:
 
   **NOTE:** some plugins might inhibit the auto-shutdown, for instance thunderbolt.
 
+**IdleInhibitSlowStartup={{IdleInhibitSlowStartup}}**
```

Comment:
@superm1 better names welcome for this.

Reply:
How about using the word threshold

Reply:
So `IdleInhibitStartupThreshold` perhaps?

Reply:
Yes

Reply:
Done.